src/harness/testcases/WINNF_FT_S_SIQ_testcase.py

```diff
@@ -16,8 +16,10 @@
 import os
 import sas
 import sas_testcase
-from util import winnforum_testcase, getRandomLatLongInPolygon, \
-  makePpaAndPalRecordsConsistent, filterChannelsByFrequencyRange
+import logging
+from util import winnforum_testcase, writeConfig,loadConfig, configurable_testcase ,getRandomLatLongInPolygon, \
+  makePpaAndPalRecordsConsistent,generateCpiRsaKeys, generateCpiEcKeys, convertRequestToRequestWithCpiSignature, \
```

Comment:
we are not doing any cpi signature conversions on this TC, could we remove the 3 cpi imports?

Reply:
Modified,   removed the cpi related methods